fastlane/Fastfile

```diff
@@ -41,17 +41,18 @@ platform :android do
 
   desc "Tag release version with latest if necessary"
   lane :tag_release_with_latest_if_needed do |options|
-    release_version = options[:release_version]
+    release_version = options[:version]
+    puts "Release version is #{release_version}"
 
-    current_branch = git_branch
     sh("git", "checkout", "tags/latest")
     latest_version = android_get_version_name(gradle_file: gradle_file_path)
```

Comment:
can we add a comment about whether we've assuming we've already updated version in gradle or not when we call this? that always trips me up